docs/styling/THEME.md

```diff
@@ -408,3 +408,48 @@ Here is an example of how your files architecture could look:
 >>>> **my-component-pres.style.scss**: component style - import style.theme.scss\
 >>>> **my-component-pres.style.theme.scss**: variables - import app-styling.scss\
 >>>> ...
+
+## Technical structure (advance)
+
+The function `apply-theme` is expected a specific structure of [SCSS Map](https://sass-lang.com/documentation/values/maps/). To facilitate its generations, the function `meta-theme-to-otter` is helping to convert an [Angular Material Theme](https://material.angular.io/guide/theming) to a compatible structure.
```

Comment:
```suggestion
The function `apply-theme` expects a specific structure of [SCSS Map](https://sass-lang.com/documentation/values/maps/). To facilitate the generation of said structure, the function `meta-theme-to-otter` converts an [Angular Material Theme](https://material.angular.io/guide/theming) into a compatible structure.
```

Reply:
@cpaulve-1A of the latter ? 
I would simplify and say "To facilitate its generation" (close to the current text) or "To facilitate the generation of this structure"